test/jasmine/tests/gl2d_click_test.js

```diff
@@ -269,12 +277,23 @@ describe('Test hover and click interactions', function() {
             pointNumber: 0
         });
 
+        // after the restyle, autorange changes the x AND y ranges
+        // I don't get why the x range changes, nor why the y changes in
+        // a different way than in the previous test, but they do look
+        // correct on the screen during the test.
```

Comment:
I think it makes sense that the axis ranges are different here than in the test above, as this is scattergl-fancy... but then why are they the same as the test above during the first `run`? This test suite baffles me...